src/Panel/Panel.tsx

```diff
@@ -22,12 +24,14 @@ const Panel = ({
       style={style}
       onClick={isClickable ? onClick : () => null}
     >
-      {title && (
-        <Title type="panelheader" className="commonPanel__title">
-          {title}
-        </Title>
-      )}
-
+      <div className={`commonPanel__titleContainer`}>
```

Comment:
This is static so don't need the function bracket 

```suggestion
      <div className="commonPanel__titleContainer">
```